src/connectors/connectStateResults.js

```diff
@@ -0,0 +1,45 @@
+const connectStateResults = (renderFn, unmountFn) => (widgetParams = {}) => ({
+  init({ instantSearchInstance }) {
+    renderFn(
+      {
+        state: undefined,
+        results: undefined,
+        instantSearchInstance,
+        widgetParams,
+      },
+      true
+    );
+  },
+
+  render({ results, instantSearchInstance, state }) {
+    const resultsCopy = Object.keys(results).reduce((acc, key) => {
+      if (key !== '_state') {
```

Comment:
Any reasons for removing the `_state` key? React InstantSearch doesn't:

![SearchResults in RIS](https://user-images.githubusercontent.com/6137112/66465075-728bf880-ea80-11e9-9601-c40a56ea7c3b.png)

Reply:
I want to avoid giving access to the helper functions here, only the values. Might be overkill to remove it, but it doesn't cost much to remove it.

Reply:
It would have been nice to align the two but I guess we can do that once we have this kind of connector in InstantSearch.js.